### DISCUSSION TITLE
Se puede comenzar con la primer practica o eso es durante el Live de mañana?

### DISCUSSION THREAD

**Discussion by TheSempaii** [Launch-X-Latam/MisionFrontEnd#38] [Q&A]
Mi pregunta es si podemos comenzar con la practica del primer tema o esos solo se trabajan durante los Lives de los Miercoles y Viernes?

**Comment by Pablo-Jaramillo** [comment]
Me parece que puedes revisarlo, después de todo esta desbloqueado, ya mañana tendrás retroalimentación.

**Comment by doguedogue** [comment]
Claro, avanza a tu ritmo si le sabes, para los que tenemos dudas lo validamos en las sesiones en vivo 🚀
